public/locations-list-map.js

```diff
@@ -164,7 +157,7 @@ function createFilterElements(filters) {
     return filterContainer;
   }
 
-  container.appendChild(ce('h4', null, ctn($.i18n('ftm-states'))));
+  container.appendChild(ce('h4', null, ctn($.i18n('ftm-administrative-region-filter', $.i18n(countries[currentCountry].administrativeRegionI18nString)))));
```

Comment:
The string here is "Filter by $1:" where $1 = "State" for US, "Province" for CA, and "Department" for France